web/src/components/form/InputField.vue

```diff
@@ -6,7 +6,7 @@
       <slot v-else-if="$slots.titleActions" name="titleActions" />
     </div>
     <slot :id="id" />
-    <div v-if="$slots.description" class="text-wp-text-alt-100 ml-1">
+    <div v-if="$slots.description" class="text-wp-text-alt-100 ml-1 mt-1">
```

Comment:
This ensures enough margin between the text and the form.

<img width="232" alt="image" src="https://github.com/user-attachments/assets/bc9ecffd-e0de-4b68-9c85-b76b33c787db" />